kolibri/plugins/learn/assets/src/actions.js

```diff
@@ -202,6 +202,14 @@ function redirectToLearnChannel(store) {
     });
 }
 
+function showChannelList(store) {
```

Comment:
I'm wondering if perhaps a better name is `updateChannelList`? We tend to use `show...` as a pattern for pages and navigation, so I'd like to avoid insinuating that there's a "Channel List" page